version.sh

```diff
@@ -3,6 +3,8 @@
 # A script to bump the version number on all Cargo.toml files etc in
 # an atomic fashion.
 
+set -e -o pipefail
```

Comment:
Ah, I only see this script now. So maybe it's not really a hassle if it's automatized, although the scripts feel a bit ad-hoc.

Reply:
It is somewhat automated, in a kinda hacky way. I think we could do a little bit better here in terms of setting up the release process steps. Not sure what big crates do in this regard. Maybe something that includes a mini-crater run or cargo-semvar-checks to help catch any last minute issues.